man/multiply.rw.Rd

```diff
@@ -0,0 +1,27 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/rw.R
+\name{multiply.rw}
+\alias{multiply.rw}
+\title{Arithmetically multiplies two reweighting factors}
+\usage{
+multiply.rw(rw1, rw2, nf1 = 1, nf2 = 1)
+}
+\arguments{
+\item{rw1, rw2}{\code{rw_orig} objects to be muplitplied}
+
+\item{nf1, nf2}{Integer. Factors that determines the number of flavours we
+have for reweighting factor 1 and 2, the default value is 1, because
+usually we compute the reweighting factors for Q: the product of
+up and down or strange and charm determinant, there are no additional
+terms in the sea determinant that have to be reweighted with the same
+rw factor. In case we compute the rw1 factor only for the up quark, we
+for example, we have to set nf1=2 to obtain the rw factor
```

Comment:
why is that though? Don't we do `det(M_new^\dag M_new)/det( (M_old)^\dag M_old )` ?

Reply:
yes, you are right, in our case this is not really the number of flavours. Should I name it as the number of determinants you wanted to reweight, or should we forget this option? I just thought that in some cases it is possible that you want to introduce the reweighting factor with an additional power.